packages/teams-js/src/public/tasks.ts

```diff
@@ -18,7 +18,8 @@ import { BotUrlDialogInfo, DialogInfo, DialogSize, TaskInfo, UrlDialogInfo } fro
 export namespace tasks {
   /**
    * @deprecated
-   * As of 2.0.0-beta.4, please use {@link dialog.open(dialogInfo: DialogInfo, submitHandler?: DialogSubmitHandler, messageFromChildHandler?: PostMessageChannel): PostMessageChannel} instead.
+   * As of 2.0.0-beta.4, please use {@link dialog.open(urlDialogInfo: UrlDialogInfo, submitHandler?: DialogSubmitHandler, messageFromChildHandler?: PostMessageChannel): PostMessageChannel} for url based dialogs
```

Comment:
developers still have to use tasks for card-based dialogs.  How do I say that this function is deprecated but still use it for card based?

Reply:
After updating to the latest SDK, we are unable to use dialogs for cards. Is the recommendation here that we revert to tasks for those? And is this temporary?
This will require reverting many of the changes made for MetaOS support, I would prefer not doing that if you'll eventually add card support back to dialogs.

Reply:
You should talk about this on teams